packages/security/src/appSecurity/AppSecurity.ts

```diff
@@ -15,8 +15,21 @@ export class AppSecurity {
     } else {
       console.warn("Security configuration is missing. The Security module will not work properly.");
     }
+    console.info("Url: ", this.internalConfig.url);
```

Comment:
I would remove that or add more info to the header and debug type.

Reply:
:+1: